File: src/main/java/no/nav/domain/pensjon/regler/beregning/penobjekter/SkattefriUforetrygdOrdiner.java
package no.nav.domain.pensjon.regler.beregning.penobjekter;

import no.nav.domain.pensjon.regler.kode.YtelsekomponentTypeCti;

/**
 * Created by N123422 on 07.11.2014.
 */
public class SkattefriUforetrygdOrdiner extends BeregningYtelseskomponent {

    

    private Integer pensjonsgrad = 0;

    public SkattefriUforetrygdOrdiner() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("SKATT_F_UT_ORDINER");
    }


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning/penobjekter/SkattefriUforetrygdOrdiner.kt
package no.nav.pensjon.regler.domain.beregning.penobjekter

import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

/**
 * Created by N123422 on 07.11.2014.
 */
class SkattefriUforetrygdOrdiner : BeregningYtelseskomponent() {
    var pensjonsgrad = 0
    override var ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.SKATT_F_UT_ORDINER
}